app/models/tag.rb

```diff
@@ -50,6 +50,8 @@ def css_class
   def valid_for?(user)
```

Comment:
We've got some semantic overloading happening. `Tag#valid_for` is also used by `Tag.all_with_filtered_counts_for` (which should probably be done by SQL rather than Ruby, possibly as a separate query as 0b67cd2de932d03027959901a5d240b8dfd53c50 did for `Tag.active.all_with_story_counts_for`). So this change to `valid_for` prohibits new users from filtering out `ask` posts.

I don't want to change the core behavior, the goal of #23 was to make sure users don't filter out the admin-only tags.

Maybe the right solution here is renaming `valid_for?` to something more like `can_be_applied_by?` and creating a similar second method named something like `can_be_filtered_by?` (I'm not in love with these names, top of my head.)